src/bin/read.rs

```diff
@@ -0,0 +1,36 @@
+extern crate parquet;
```

Comment:
I actually prefer having them separate in `bin` directory for now: `dump-schema` is for inspecting schema and `read`/`cat` for listing records in a file. Then later we can move those files into a subproject `parquet-rs-tools`. Would you like this approach?

Reply:
OK. Sounds fine. 

Another problem: there's name collision since read is already taken by the Shell built-in command, and if you install the binary via `cargo install`, you'll not be able to use this one.

Reply:
Aaa, okay. I will rename it to something that does not have conflicts, thanks!